kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -180,4 +188,20 @@ private Object convertLogical(Object kafkaConnectObject,
         LogicalConverterRegistry.getConverter(kafkaConnectSchema.name());
     return converter.convert(kafkaConnectObject);
   }
+
+  private Double convertDouble(Object kafkaConnectObject, Boolean shouldConvertSpecialDouble) {
+    double kafkaConnectDouble = (double)kafkaConnectObject;
```

Comment:
For null pointer safety, recommend casting to boxed `Double` instead of `double`. Also, recommend having param be `Double kafkaConnectDouble`, and forcing casting to happen in the invoking method rather than inside this method.